src/Layer/TiledGeometryLayer.js

```diff
@@ -371,14 +372,16 @@ class TiledGeometryLayer extends GeometryLayer {
         // The induced geometric error is much too large and distorts the SSE
         const nodeLayer = node.material.getElevationLayer();
         if (nodeLayer) {
-            const currentTexture = nodeLayer.textures[0];
-            if (currentTexture && currentTexture.extent) {
-                const offsetScale = nodeLayer.offsetScales[0];
-                const ratio = offsetScale.z;
-                // ratio is node size / texture size
-                if (ratio < 1 / Math.pow(2, this.maxDeltaElevationLevel)) {
-                    return false;
+            for (const [extent, texture] of nodeLayer.textures.entries()) {
+                if (texture) {
+                    extent.offsetToParent(texture.coords, offsetScale);
```

Comment:
You can avoid calling `offsetToParent` by maybe adding a new method on `Extent`, something that only compute the `z` part, as it is the only thing used here.